sim/warrior/devastate.go

```diff
@@ -53,6 +89,9 @@ func (warrior *Warrior) registerDevastateSpell() {
 				if spellEffect.Landed() {
 					if !warrior.ExposeArmorAura.IsActive() {
 						warrior.SunderArmorDevastate.Cast(sim, spellEffect.Target)
+						if hasGlyph {
```

Comment:
I might be wrong, but isn't this handled in sunder_armor.go lines 36 and 57? Could be duplicate logic here.

Reply:
Oh! You're absolutely right, thank you!